README.md

```diff
@@ -83,6 +114,31 @@ cla_public:
 
 where the `context` directory is set to the root of the cla_public directory.
 
+## Using Kubernetes
+
+Read the following if you want to use Kubernetes from your local development environment.
+
+### Setup kubectl
+
+You'll need to install and configure `kubectl` CLI tool to interact with Kubernetes. There are [instructions on kubectl configuration](https://ministryofjustice.github.io/cloud-platform-user-docs/01-getting-started/001-kubectl-config/) in the Cloud Platform User Guide.
+
+### Kubernetes namespaces and admin role
+`cla_public` has two namespaces (environments):
+
+- [laa-cla-public-staging](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-cla-public-staging)
+- [laa-cla-public-production](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-cla-public-production)
+
+To gain the `ClusterRole - admin` role, you must be a member of the GitHub team `laa-get-access` - [find out more about roles](https://ministryofjustice.github.io/cloud-platform-user-docs/01-getting-started/002-env-create/#01-rbacyaml).
+
+### Authenticating with AWS ECR repository
+Docker images are stored in AWS ECR. To authenticate with the `cla_public` repository, fetch the credentials by typing the following:
+
+```
+kubectl --namespace laa-cla-public get secrets -o yaml
```

Comment:
❓ Would we need to use either `laa-cla-public-staging` or `laa-cla-public-production` here?

Reply:
`laa-cla-public-staging` - good spot.